docs/serving/load-balancing/README.md

```diff
@@ -15,7 +15,7 @@ You can turn on Knative load balancing, by placing the _Activator service_ in th
 Activator pods are scaled horizontally, so there may be multiple Activators in a deployment. In general, the system will perform best if the number of revision pods is larger than the number of Activator pods, and those numbers divide equally.
 <!--TODO(#2472): Add better documentation about what the activator is; explain the components of load balancing; maybe add a diagram-->
 
-Knative assigns a subset of Activators for each revision, depending on the revision size. More revision pods will mean a greater number of Activators for that revision.
+Knative assigns a subset of Activators for each revision, depending on the revision size and the configured capacity per Activator. More revision pods will mean a greater number of Activators for that revision.
```

Comment:
being a pedant, is this also dependent on tbc?

Reply:
Oh I guess I should strip both out again as those concepts have not yet been introduced.